src/lib.rs

```diff
@@ -225,8 +225,25 @@ impl<T> Slab<T> {
     /// # use slab::*;
     /// let slab: Slab<i32> = Slab::new();
     /// ```
-    pub fn new() -> Slab<T> {
-        Slab::with_capacity(0)
+    #[cfg(not(slab_no_const_vec_new))]
+    pub const fn new() -> Self {
```

Comment:
The fact that const depends on the version should probably be documented.

Reply:
Updated docs.